CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
 [//]: # (current developments)
 
+## 23.10.0 (2023-10-30)
+
+## ✨ Special announcement ✨
+
+This is an announcement about an important change in conda's functionality:
+
+### **With this 23.10.0 release we are changing the default solver of conda to [`conda-libmamba-solver`](https://conda.github.io/conda-libmamba-solver/)!** 🥳 🚀
+
+The previously "classic" solver is based on [pycosat](https://github.com/conda/pycosat)/[Picosat](http://fmv.jku.at/picosat/) and will remain part of conda for the foreseeable future, a fallback is possible and available.
+
+### Why are we switching the solver?
+
+In short: to make conda faster and more accurate.
+
+A "solver" is the core component of most package managers; it calculates which dependencies (and which version of those dependencies) to install when a user requests to install a package from a package repository. To address growth-related challenges within the conda ecosystem, the conda maintainers, alongside partners Anaconda, Quansight and QuantStack, introduced [a new conda dependency solver based on the Mamba project](https://mamba.readthedocs.io) in December 2022.
+
+Since July 2023, that [`conda-libmamba-solver`](https://github.com/conda/conda-libmamba-solver) plugin has been included in and automatically installed with all major conda ecosystem installers (miniforge, miniconda, mambaforge and Anaconda Distribution), _with the default solver configuration unchanged_.
```

Comment:
Might be me but this could be read like the miniconda distribution and friends will not switch the default solver in the future either? What about adding a follow up sentence that emphasizes that starting with conda 23.10, this won't be the case anymore?